tests/e2e/util.go

```diff
@@ -16,3 +31,207 @@ func ReadModuleTemplate(filepath string) (*v1beta2.ModuleTemplate, error) {
 	err = yaml.Unmarshal(moduleFile, &moduleTemplate)
 	return moduleTemplate, err
 }
+
+func ExecuteKymaDeployCommand() error {
+	deployCmd := exec.Command("kyma", "alpha", "deploy")
+	deployOut, err := deployCmd.CombinedOutput()
+	if err != nil {
+		return errKymaDeployCommandFailed
```

Comment:
Should not we return the original error too? Now if error occurs, the context will be lost and logs will contain only the `failed to run kyma alpha deploy ` line.

I suggest this way
```go
fmt.Errorf("%w: %v", errKymaDeployCommandFailed, err)
```
as a general strategy for error handling in test suites.